haproxy/values.yaml

```diff
@@ -0,0 +1,28 @@
+replicaCount: 1
+
+image:
+  repository: docker.io/haproxytech/haproxy-debian
+  tag: 2.1
+  pullPolicy: IfNotPresent
+
+ingress:
+  enabled: false
+
+service:
+  port: 8080
+  type: ClusterIP
+
+haproxy:
+  config: tls
+  frontend:
+    port: 30636
+  backend: example.org
```

Comment:
I would comment out the `backend` as it won't work in any case and could lead to leaked secrets to example.org if things go really wrong.